v8/prettybasic/templates/authors.tmpl

```diff
@@ -0,0 +1,25 @@
+{#  -*- coding: utf-8 -*- #}
+{% extends 'prettybasic.tmpl' %}
```

Comment:
The base file *should* be called `base.tmpl`. If you don’t do that, you can end up with pages looking the wrong way because of missing templates.

Reply:
It seems I had misunderstood your yesterday's request. I am going to undo the changes.